app/controllers/api/v2/config_groups_controller.rb

```diff
@@ -23,6 +23,7 @@ def show
       def_param_group :config_group do
         param :config_group, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :puppetclass_ids, Array
```

Comment:
if we had a proper test [here](https://github.com/theforeman/foreman/blob/develop/test/controllers/api/v2/config_groups_controller_test.rb#L13) and verify that puppet classes were added we'd probably notice the missing parameter sooner, would you mind extending it?